src/main/java/eu/bbmri_eric/negotiator/governance/resource/dto/ResourceCreateDTO.java

```diff
@@ -0,0 +1,21 @@
+package eu.bbmri_eric.negotiator.governance.resource.dto;
+
+import jakarta.validation.constraints.NotNull;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+@Getter
+@Setter
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+public class ResourceCreateDTO {
+
+  @NotNull private String name;
+  private String description;
+  @NotNull private String sourceId;
+  @NotNull private Long organizationId;
```

Comment:
add AccessFrom ID and Discovery service id. If not presenet then use 1 by default